.github/workflows/trivy.yml

```
@@ -0,0 +1,20 @@
+name: build
+on:
+  push:
+    branches:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)